### DISCUSSION TITLE
[BUG]  Not decrypting

### DISCUSSION THREAD

**Discussion by RodAlc24** [zeapo/Android-Password-Store#2868] [Q&A] [answered]
### Describe the bug

When I try to decrypt a password the app fails with a "Wrong password" error. 
This only happens if it where encrypted in my computer:

|               | encrypted in pc | encrypted in app |
|---------------|:---------------:|:----------------:|
| decrypt in pc | ok              |               ok |
|decrypt in app | Wrong password  |               ok |

I know the password is correct. I have tried with many different keys but it keeps failing. 

### Steps to reproduce

1. Create a new entry in the pc.
2. Synchronize the app repository.
3. Try to decrypt the entry (write the gpg password)

### Expected behavior

Decrypt all passwords, not only the generated in the app.

### Device information

- Device: Huawei HRY-LX1
- OS: Android 10
- App version: Snapshot 2.0.0 (nonFree)

### Additional context

[log.txt](https://github.com/android-password-store/Android-Password-Store/files/14022533/log.txt)

This log is the one of my last try, with a NIST curve as mentioned in #2461.

Maybe I am encrypting it wrong in my pc.

**Comment by msfjarvis** [comment]
The log indicates you're still using an AEAD-enabled key, so you got something wrong in the process. I would suggest trying again from scratch.

**Reply by RodAlc24** [reply]
Thank you for responding so quickly. I deduce from your answer that the error is in generating the GPG key, what type of key should I use? 
Maybe I'm getting it wrong, sorry for the inconvenience.

**Reply by msfjarvis** [reply, marked as answer]
Any key you generate will work fine, you only need to edit it to drop the AEAD bit: https://wiki.archlinux.org/title/GnuPG#Disable_unsupported_AEAD_mechanism

**Reply by RodAlc24** [reply]
Thanks!! It worked.
